packages/pf-codemods/README.md

```diff
@@ -112,6 +112,24 @@ Out:
 <Alert component="h4" />
 ```
 
+### applicationLauncher-deprecate [(8836)](https://github.com/patternfly/patternfly-react/pull/8836)
+
+We've deprecated the `ApplicationLauncher` compoenents. A fix will update code to point to the new deprecated import, but we recommend replacing with `Dropdown` or `Select`
```

Comment:
Just a typo. Otherwise looks good!
```suggestion
We've deprecated the `ApplicationLauncher` components. A fix will update code to point to the new deprecated import, but we recommend replacing with `Dropdown` or `Select`
```